modules/stochastic_tools/test/tests/distributions/truncated_normal.i

```diff
@@ -0,0 +1,59 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 1
+  ny = 1
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Distributions]
+  [./normal_test]
+    type = TruncatedNormalDistribution
+    mean =  0
+    standard_deviation = 1
+    lower_bound = -10
```

Comment:
It looks like your results are almost identical to the normal distribution.  I'd use tighter bounds so that you actually have a measurable difference that you can check against a known solution.